### PR TITLE
[DDW-1125] Analytics - part 4 - environment configs

### PR DESCRIPTION
**Pull request by marcin-mazurek**
This PR implements the final Matomo config.

## Todos

- [ ] TODO

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test

---

### Basics

- [ ] PR assigned to the PR author(s)
- [ ] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [ ] If there are UI changes, Alexander Rukin assigned as an additional reviewer
- [ ] All visual regression testing has been reviewed (assign `run Chromatic` label to PR to trigger the run)
- [ ] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [ ] PR link is added to a Jira ticket, ticket moved to In Review
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR contains screenshots (in case of UI changes)
- [ ] CHANGELOG entry has been added to the top of the appropriate section (_Features_, _Fixes_, _Chores_) and is linked to the correct PR on GitHub
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality

- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with typescript types
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing

- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review

- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/config/analyticsConfig.ts [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
 
-export const ANALYTICS_API_ENDPOINT =
-  'https://daedalusqa2.matomo.cloud/matomo.php';
+export const ANALYTICS_API_ENDPOINT = 'https://matomo.cw.iog.io/matomo.php';
 export const PRIVACY_POLICY_LINK =
   'https://static.iohk.io/terms/iog-privacy-policy.pdf';
-export const DEV_MODE_SITE_MAP_ID = 1;
+
+// ID used when Daedalus is launched from nix-shell
+export const DEV_MODE_SITE_MAP_ID = 11;
```
Comment (DominikGuzei) [inline source/renderer/app/config/analyticsConfig.ts:8]: Is it possible to define the ID as a string? The magic numbers are ugly IMO

Comment (marcin-mazurek) [reply source/renderer/app/config/analyticsConfig.ts:8]: `MatomoTracker` expects a number as site ID, so I would have to cast those strings to a number later on, which introduces room for error (NaN). Could we stick to number? :)

**Review by lucas-barros — APPROVED** [review]

File: source/renderer/app/config/analyticsConfig.ts (hunk 1 above)
Comment (DominikGuzei) [reply source/renderer/app/config/analyticsConfig.ts:8]: Ok, that's sad but lets do it then 👍

**Review by DominikGuzei — APPROVED** [review]

**Review by gabriela-ponce — APPROVED** [review]

**Review by miorsufianiohk — APPROVED** [review]
LGTM. Great work @marcin-mazurek 👍 . Tested on 22172

**Pull request closed without merging**